src/backend/main.ts

```diff
@@ -358,6 +358,8 @@ if (!gotTheLock) {
       OverlayApp.toggleIntercept
     )
 
+    globalShortcut.register('F11', () => backendEvents.emit('toggleFullscreen'))
```

Comment:
I wonder if we could just use a local shortcut for that that is created after the browser game was created. Although F11 should be fine, using GlobalShortcut will override all F11 shortcuts on all apps that uses it while HyperPlay is opened. That might not be a big deal though, but a local shortcut could work better.

Reply:
great idea. i didn't think about this. just changed it